qa-niaid.planx-pla.net/manifest.json

```diff
@@ -7,27 +7,27 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.03",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.04",
     "audit-service": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/audit-service:0.1.0",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
-    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2021.03",
+    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2021.04",
     "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:4.28.1",
```

Comment:
let's deploy Fence 4.29.1 - i'm not sure if it's in ECR already

Reply:
let me check if the fence image is in ECR

Reply:
Yes the image is already in ECR, making a commit for desired change